### ISSUE TITLE
Difference between tuner objective and loss from model optimizer?

### ISSUE THREAD

**Issue by barnardp** [keras-team/keras-tuner#440]
Hi, I am wondering what is the difference between the objective metric used by the tuner and the loss metric used by the optimizer? For example, I am currently optimizing a DNN using adam optimizer with a huber loss function. Most examples I've seen till now use both 'mse' for the optimizer loss and the objective metric within the tuner. If I use objective = 'mse' in the tuner will this overwrite the loss metric of my optimizer? Any advice would be much appreciated :)

Kindest regards, 
Pieter
